packages/nimble-components/src/table-column/number-text/models/unit-prefix.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * A prefix that represents a scaling factor when applied to a base unit
+ */
+export class UnitPrefix {
+    public constructor(
+        public readonly factor: number,
+        public readonly text: string
+    ) {}
+}
+
+// The same prefixes are used for English, French, German, Japanese, and Chinese (all currently supported languages).
```

Comment:
I confirmed with Eiko from localization that all the abbreviated units for voltage (e.g. mV, cV, dV, kV, ...) were the same as English for the other four languages we support. I don't think that is unique to voltage. From my experimenting, I see that `Intl.NumberFormat` does use "kg" for kilograms in Chinese:
![image](https://github.com/ni/nimble/assets/7282195/a711ae37-cb77-4efa-9f9d-d939ae83bbef)
Did you maybe pass "gram" as the unit? `Intl.NumberFormat` does weird things (like, for some languages, giving "1k" instead of "1,000") if you only pass the base unit.

I did see that using Google translate for "kg" in Chinese produced "公斤". However, it looks like that's just the the translation for the word "kilogram". The [Wikipedia page for SI units in Chinese](https://zh.wikipedia.org/wiki/%E5%9B%BD%E9%99%85%E5%8D%95%E4%BD%8D%E5%88%B6) shows the same unit abbreviations and prefix symbols as the English page. That by itself doesn't guarantee that they use SI representations, but the page also has a picture of a highway sign with distances in "km". In any case, based on what `Intl.NumberFormat` does and what Eiko said, I'm pretty confident that we're using reasonable (if not exclusively used) representations.